client/lib/purchases/index.js

```diff
@@ -20,9 +20,11 @@ import {
 	isJetpackPlan,
 	isPlan,
 	isTheme,
+	isJetpackProduct,
```

Comment:
It's very common to sort these alphabetically. Could move it just below `isJetpackPlan`?